### DISCUSSION TITLE
The output of %l is the same as the name that I entered

### DISCUSSION THREAD

**Discussion by Al-Sharabi** [chubin/wttr.in#857] [General]
Hello, I noticed that `https://wttr.in/{place}?format=%l` returns the same text that I entered in place, and not the actual location that `https://wttr.in/{place}` shows. Like for example, if I `curl -s 'wttr.in/Taizz?format=%l'` I see the output as `Taizz` while when I `curl -s 'wttr.in/Taizz'` I get the actual location name at the bottom which is  `Ta'izz Governorate, Yemen [13.4115414,43.5570871]` . Can you make it so that %l format returns the actual location instead of returning the name that I just entered? Or maybe add another format which returns the correct/official name of the place. Thanks.

**Comment by hsolatges** [comment]
I am looking for it too!
Is there a way to get the adress of the weather station ? 🙏
